build/release/Makefile

```diff
@@ -57,7 +57,7 @@ endif
 DOCKER_REGISTRY ?= rook
 QUAY_REGISTRY ?= quay.io/rook
 REGISTRIES ?= $(DOCKER_REGISTRY) $(QUAY_REGISTRY)
-IMAGES ?= rook toolbox
+IMAGES ?= rook toolbox flexvolume
```

Comment:
I'd like to avoid creating a new image to install flex. we should use the same rook image, add a new verb that installs the flex volume.